homework/tests/example.spec.js

```diff
@@ -1,31 +1,4015 @@
 // @ts-check
 const { test, expect } = require('@playwright/test');
 
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+});
+
 const data = [
   'Prototype',
-  '1',
```

Comment:
You have commented all the required data for test cases [1-9] and only tested "Prototype" build. If we read the homework description, it is described that:
1. The calculator application consists of 10 different builds. Prototype version has no known flaws. 
2. The (1-9) builds contain bugs
So we can go with 2 approaches: 
1. Include prototype build along with 1-9 builds just to be sure that there are no bugs as written in the description
2. Test only 1-9 builds

With your approach only prototype build is tested and no other builds are included. Incorrect strategy is selected.

I believe you didn't come up with solution how to re-use single test all the builds from 1 to 9 or misunderstood the homework.

Reply:
I was testing only 1 build at the time (commented all other builds apart "Prototype", run tests, report bugs, repeat with build 1, then 2 and so on), so that flaws can be seen clearly. With that intention I left only 1 build uncommented.

To test all cases there is an easy fix of just uncommenting all builds

Reply:
Each test should be executed on each build and this is the main idea that your code is able to run on each build. I can see that there are 116 test-cases for just a single build so in total you would have around 1k which could cause problems when trying to identify issues. For this issue we have build version + test-case description which would help to identify the issue easily. Attaching a screenshot as an example. And again, try to improve your code and remove repetitive scenarios.
![image](https://user-images.githubusercontent.com/73665598/180757482-9ef99634-ad3b-461e-9495-aeec32ec4735.png)